### DISCUSSION TITLE
Does anyone know about the serverSocket web local server?

### DISCUSSION THREAD

**Discussion by Jason0247** [airsdk/Adobe-Runtime-Support#2942] [Q&A]
A long time ago, I saw someone post about a small website server implemented using Air, which can dynamically create any local folder as the target of the website server.
Unfortunately, I didn't bookmark that link at the time, and now I can't find it. Do you know about this?
thanks

**Comment by 2jfw** [comment]
The only thing which comes close is: https://github.com/airsdk/Adobe-Runtime-Support/discussions/2907#discussioncomment-7533389
However, it's from a recent post and not that long ago..

**Comment by FischMich** [comment]
Maybe: https://github.com/leopoldodonnell/airhttp

**Reply by FischMich** [reply]
https://github.com/xabip/as3-server

**Comment by Jason0247** [comment]
Thank you all!

**Reply by 2jfw** [reply]
Out of interest, what you are trying to do exactly and is one of the links from @FischMich what you were looking for (/*curious*)?

**Reply by Jason0247** [reply]
I want to use KrPano to run in a local environment, which requires server support. Therefore, I am considering writing a simple HTTP server internally and loading and displaying it using a method like 127.0.0.1.
Windows can already be implemented, but unfortunately, Android cannot. For some reason, Android cannot display panoramic views, and buttons in HTML can be displayed.

**Reply by 2jfw** [reply]
Alright, I see - thanks for the details :)
